download_script.py

```diff
@@ -0,0 +1,29 @@
+
+# author: MDS-2021-22 block3 group21
+# date: 2021-11-19
+
+"""Downloads data as a csv file from online to a local filepath via a URL.
+Usage: download_script.py --url=<url> --filepath=<filepath> 
+ 
+Options:
+--url=<url>             URL from where to download the data (must be in standard csv format)
+--out_file=<out_file>   Path (including filename) of where to store the csv file locally
```

Comment:
Discrepancy between usage and options. Usage refers to filepath while options refers to out_file.